### PR TITLE
Merge release/4.2 to develop

### PR DESCRIPTION
**Pull request by anwar6953**
There were conflicts in the following files that were fairly straightforward to resolve:
ETLMapReduce.java
MapReduceBatchContext.java
cdap-ui/pom.xml

### CODE REVIEW

File: cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/mapreduce/ETLMapReduce.java [hunk 1]
```diff
@@ -202,7 +202,6 @@ public void initialize() throws Exception {
     for (String sourceName : phaseSpec.getPhase().getSources()) {
       try {
         BatchConfigurable<BatchSourceContext> batchSource = pluginInstantiator.newPluginInstance(sourceName, evaluator);
-
```
Comment (CuriousVini) [inline cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/mapreduce/ETLMapReduce.java:205]: nit: no need to remove this line

Comment (anwar6953) [reply cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/mapreduce/ETLMapReduce.java:205]: It was removed on release/4.2:
https://github.com/caskdata/cdap/blob/release/4.2/cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/mapreduce/ETLMapReduce.java?utf8=%E2%9C%93#L204

**Comment by ajainarayanan** [comment]
👍  UI Change LGTM

File: cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/mapreduce/ETLMapReduce.java (hunk 1 above)
Comment (CuriousVini) [reply cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/mapreduce/ETLMapReduce.java:205]: I see

**Comment by CuriousVini** [comment]
lgtm for `ETLMapReduce` and `MapReduceBatchContext `

**Pull request merged**